src/foam/nanos/rope/ROPE.js

```diff
@@ -48,13 +48,13 @@ foam.CLASS({
     },
     {
       name: 'crudMap',
-      class: 'Map',
-      javaType: 'java.util.Map<String, java.util.Map<String, java.util.List<String>>>'
+      class: 'Object',
```

Comment:
Does this work?

```suggestion
      class: 'FObjectProperty',
      of: 'foam.nanos.rope.CRUDMap',
```

Similar suggestion for `relationshipMap` below.

Reply:
yes, changed